docs/reference/ccr/getting-started.asciidoc

```diff
@@ -304,37 +253,36 @@ POST /server-metrics-follower/_ccr/unfollow
 ====
 
 [[ccr-getting-started-auto-follow]]
-==== Automatically create follower indices
-Create <<ccr-auto-follow,auto-follow patterns>> to automatically follow time
-series indices that are periodically created in a remote cluster (such as daily
-{beats} indices).
+==== Create an auto-follow pattern to replicate time series indices
+You use <<ccr-auto-follow,auto-follow patterns>> to automatically create new
+followers for rolling time series indices. Whenever the name of a new index on
+the remote cluster matches the auto-follow pattern, a corresponding follower
+index is added to the local cluster.
 
-With an auto-follow pattern, you reference the
-<<ccr-getting-started-remote-cluster,remote cluster>> connected to your
-local cluster. You must also specify a collection of patterns that match the
-indices you want to automatically follow.
+An auto-follow pattern specifies the remote cluster you want to replicate from,
+and one or more index patterns that specify the rolling time series indices you
+want to replicate. When the auto-follow pattern matches the specified index
+pattern, {es} creates new time series indices on the remote cluster.
```

Comment:
I think I confused the issue with my comment. I'd just remove this second sentence.

```suggestion
want to replicate.
```